xbmc/pvr/recordings/PVRRecording.h

```diff
@@ -535,6 +547,7 @@ class CPVRRecording final : public CVideoInfoTag
   std::string m_strProviderName; /*!< name of the provider this recording is from */
   int m_iClientProviderUniqueId =
       PVR_PROVIDER_INVALID_UID; /*!< provider uid associated with this recording on the client */
+  RecordingMediaType m_mediaType; /*!< The type of media this recording is */
```

Comment:
Should be initialized with PVR_RECORDING_MEDIA_TYPE_UNKNOWN.

Reply:
Not relevenat, media type will always be known.

Reply:
anyway, every member should be initialized,  with TYPE_RECORDING in this case

Reply:
True, I've added that default value.